mebeim

I had written a perfect program, when all of a sudden... a cosmic ray was enough to pwn my entire system :(

Note: remote container runs in Ubuntu 24.04.3 LTS host with kernel 6.14.0-1012-aws

Hint: as you may have noticed the app runs via pypy3. Did you know PyPy JITs hot Python loops to x86-64 code? That could be interesting to play with...
